desktop/hardened-desktop/enhanced-container-isolation/index.md

```diff
@@ -79,10 +79,10 @@ For this to take effect:
 
 When Enhanced Container Isolation is enabled, users see that containers run within a Linux user namespace. 
 
-To check, run:
+To check, first run:
 
 ```
-$ docker run --rm alpine cat /proc/self/uid_map 
+$ docker run -it --rm alpine cat /proc/self/uid_map 
```

Comment:
the `-it` flag is only needed if you want to start an interactive shell in the container. In this case, if we pass the entrypoint parameters to `docker run`, they are not needed.